src/components/Todo/Todo.js

```diff
@@ -62,6 +71,8 @@ const Todo = ({ id, status, onStatusChange, onDelete, onUpdateText, children, ..
     cancelEdit()
   }
 
+  const handleDetailsTodo = () => onClickDetailsTodo(id)
```

Comment:
Siguiendo la convención, falta el nombre del evento en el handler, y el evento en el listener va al final

```suggestion
  const handleDetailsTodoClick = () => onDetailsTodoClick(id)
```